api/python/tests/integration/test_packages.py

```diff
@@ -347,6 +347,22 @@ def test_local_set_dir(tmpdir):
     # todo nested at set_dir site or relative to set_dir path.
     assert (bazdir / 'baz').resolve().as_uri() == pkg['my_keys/baz'].physical_keys[0]
 
+    # Verify ignoring files in the presence of a dot-quiltignore
+    with open('.quiltignore', 'w') as fd:
+        fd.write('foo\nbar')
```

Comment:
tiny nit: can you do

```
fd.write('foo')
fd.write('bar')
```
less performant but more readable

Reply:
Actually it should be:

```
fd.write('foo\n')
fd.write('bar')
```

`write` doesn't automatically line break between writes ([ref](https://stackoverflow.com/questions/2918362/writing-string-to-a-file-on-a-new-line-every-time)), which kind of makes sense.